app/assets/stylesheets/patternfly_and_overrides.scss

```diff
@@ -0,0 +1,347 @@
+@import "mixin";
+@import "colors";
+
+$navbar-default-bg: $primary_color;
+
+.fact-col{
+  white-space: normal;
+  max-width: 500px;
+}
+
+
+
+
+.control-label{
+  text-align: left;
+  padding-left: 0px;
+  padding-right: 5px;
+}
+
+div.spinner{
+  float: left;
+}
+
+.spinner-two-pane{
+  top: 50%;
+  position: absolute;
+  left: 45%;
+}
+
+.select2-arrow > b{
+  margin: -4px 0 0 0;
+}
+
+#main {
+  padding: 0 40px;
+}
+
+fieldset > div.col-md-1 {
+  padding-left: 0px;
+}
+
+.ms-container .ms-selectable li.ms-elem-selectable, .ms-container .ms-selection li.ms-elem-selection{
+  font-size: 12px;
+}
+
+.ms-filter{
+  height: 20px;
+}
+
+.children_fields{
+  padding-left: 20px;
+}
+
+.form-control{
+  display: inline-block  //font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+}
+
+.select2-container .select2-choices .select2-search-field input, .select2-container .select2-choice, .select2-container .select2-choices .select2-chosen{
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+  line-height: 2.2;
+}
+
+.navbar-editor > span > .select2-container .select2-choices .select2-search-field input, .select2-container .select2-choice, .select2-container .select2-choices .select2-chosen {
+  height: 26px;
+
+}
+
+.select2-chosen, .select2-choice > span:first-child, .select2-container .select2-choices .select2-search-field input{
+  padding: 0 0;
+}
+
+.select2-chosen, .select2-choice > span:first-child, .select2-container .select2-choices .select2-search-field input{
+  padding-left: 6px;
+}
+
+textarea.form-control{
+  padding-top: 0px;
+  padding-bottom: 0px;
+}
+
+td i{
```

Comment:
+1 to replacing all `i` icons with `span`. `i` originally meant italic, and in [HTML5](http://www.w3.org/TR/html5/text-level-semantics.html#the-i-element) means this text has a different tone. The usage for icons is an abuse of the element simply because it is short, starts with the same letter and is not used very often.